src/App.css

```diff
@@ -67,6 +86,11 @@
     inset -2px -2px 4px rgba(0, 0, 0, 0.1);
 }
 
+.game-status-text {
+  color: black;
+  font-weight: bold;
+}
+
```

Comment:
FE changes are greatly helped by including a video recording in the PR. This way users don't need to run the branch locally to know how the UI changes, esp styling, appear. You don't need to do this now but its something that looks really great on a client project.

Reply:
oh good idea!